package.json

```diff
@@ -1,14 +1,22 @@
 {
-  "name": "node_sdk_prototype",
+  "name": "@co-digital/node_api_sdk",
   "version": "1.0.0",
-  "description": "Node SDK Prototype",
+  "description": "A Node api sdk Node.JS applications in CO Digital.",
+  "homepage": "https://github.com/cabinetoffice/node-api-sdk#README.md",
```

Comment:
Add `types` field

Reply:
"repository": {
    "type": "git",
    "url": "https://github.com/cabinetoffice/api-sdk.git"
  },

This is within the package already - is this ok?